src/tls.rs

```diff
@@ -58,6 +58,13 @@ use std::{
     io::{BufRead, BufReader},
 };
 
+/// Represents a X509 certificate revocation list.
+#[cfg(feature = "__rustls")]
+pub struct Crl {
```

Comment:
Just asking to learn, is it more common to call these CRLs? If someone saw the type name, would they know what it was, or does the longer name from rustls `CertificateRecovationList` pull its weight (for being longer)?

Reply:
For those who have experience in working with certificates the name is obvious. The long name will be more understandable for others. I will rename.